### DISCUSSION TITLE
Conversion to https://podman.io/

### DISCUSSION THREAD

**Discussion by Corshi** [AbdBarho/stable-diffusion-webui-docker#279] [Ideas]
https://podman.io/ is a replacement for docker, build from zero to fully copy docker, to the point that everything should work identically to it just by **alias docker=podman**.
People are turning against docker because of the monetization methods.

So, because I do not know how to operate docker myself, I do not know all the core docker files to edit. Really all that must by done, is create alternative version of docker files, but just running `replace` in tool like notepad++ to change all `docker` references to `podman`.

**Comment by AbdBarho** [comment]
I have not worked with podman before, but I think there is nothing in this repo that is docker specific?

the synatx file perhaps: https://github.com/AbdBarho/stable-diffusion-webui-docker/blob/78c90e543599b04ba51d38fc9696775a63f4a0f7/services/AUTOMATIC1111/Dockerfile#L1

and the commands used for running
```
docker compose --profile invoke up --build
```

Feel free to try and see if it works with podman.

If it works with no change we can add it to the docs.
